sw/device/rom_exts/rom_ext_start.S

```diff
@@ -117,12 +119,115 @@ _rom_ext_interrupt_vector:
   .option norelax
 
 /**
- * Entry point after reset. This symbol is jumped to from the handler
- * for IRQ 32.
- *
- * At the moment, this should just fault, until we have a CRT we can use.
+ * Entry point. This symbol is jumped to from the handler the Mask ROM
+ * (execution handover to ROM_EXT).
```

Comment:
nit: comment not clear.

Reply:
I have changed the comment, please let me know if it is clearer now.

Reply:
I assume this is fine, because you have approved the PR in the review below. Resolving it.